.github/CODEOWNERS

```
@@ -0,0 +1,10 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# More details are here: https://help.github.com/articles/about-codeowners/
+
+# The '*' pattern is global owners.
+
+# Order is important. The last matching pattern has the most precedence.
+
+/.*  @vbrazo
```

@vzvu3k6k wanna help me on making this repository even better?

---

@vbrazo Sure!

```suggestion
/.*  @vbrazo @vzvu3k6k
```

Feel free to commit this if you need.